.pre-commit-config.yaml

```diff
@@ -1,4 +1,18 @@
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.292
+    hooks:
+      - id: ruff
+        name: Check ruff for zwave_js_server
+        files: ^(zwave_js_server)/.+\.py$
+        args: [--fix, --exit-non-zero-on-fix]
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.0.292
+    hooks:
+      - id: ruff
+        name: Check ruff for scripts and test
+        files: ^(scripts|test)/.+\.py$
+        args: [--fix, --exit-non-zero-on-fix, --select, I001, D210, D400, D403, D415]
```

Comment:
What's the difference between these two hooks? Are we more forgiving for the tests and scripts?